docs/metrics-reporting.md

```diff
@@ -0,0 +1,31 @@
+# Set up metrics reporting in custom module
+1. Add the following dependencies to your Cargo.toml
+```[dependencies]
+cb-metrics.workspace = true
+prometheus.workspace = true
+lazy_static.workspace = true```
+2. Import the necessary dependencies:
+```use cb_metrics::sdk::MetricsProvider;
+use lazy_static::lazy_static;
+use prometheus::{IntCounter, Registry};```
+3. Lazy load the prometheus registry and custom metric (for example a counter):
+```// You can define custom metrics and a custom registry for the business logic of your module. These
```

Comment:
this is not formatted properly.

Make sure that it is rendered properly and you can include the ```rust tag so that there is syntax highlights